apps/terminal/models/component/terminal.py

```diff
@@ -160,7 +160,7 @@ def service_account(self):
 
     def delete(self, using=None, keep_parents=False):
         if self.user:
-            setattr(self.user, SKIP_SIGNAL, True)
+            setattr(self.user, OP_LOG_SKIP_SIGNAL, True)
             self.user.delete()
         self.name = self.name + '_' + uuid.uuid4().hex[:8]
         self.user = None
```

Comment:
There don't seem to be any major irregularities or significant issues with the provided code snippet that need to be optimized at this point.